lib/proxy/plugin_initializer.rb

```diff
@@ -238,7 +241,15 @@ def start_services(services, container)
 end
 
 module ::Proxy::DefaultSettingsLoader
-  def load_settings(main_plugin_settings = {})
+  def load_plugin_settings
+    load_settings({}, :log_used_settings)
```

Comment:
What about naming `log_plugin_settings` vs. `log_user_settings`.

Reply:
The need for slightly different logging  reflects one of the differences between plugins and providers -- the latter do not provide public api. Not sure what the rationale behind `log_user_settings` name is...

Reply:
Sry, typo, I was suggesting renaming `log_used_settings` to `log_plugin_settings`

Reply:
This would mirror the `load_plugin_settings`/`load_provider_settings` convention

Reply:
`log_used_settings` is/can be overridden by modules/provides, I'd like to keep the name as is.